targets/nintendoswitch.s

```diff
@@ -0,0 +1,50 @@
+.section .text.jmp, "x"
+.global _start
+_start:
+        b start
+  .word _mod_header - _start
```

Comment:
Can you format this file? Again, I don't care too much about how it's exactly formatted but right now the indentation is a bit messy.

Reply:
@racerxdl this is the last one.

Reply:
omg, that file is full broken. Fixed now.